javascript/sdk/package.json

```diff
@@ -28,8 +41,7 @@
     "eslint": "^8.0.0",
     "mocha": "^10.0.0",
     "prettier": "^2.0.0",
-    "ts-node": "^10.0.0",
-    "typescript": "^5.7.3"
+    "ts-node": "^10.0.0"
```

Comment:
I think we can drop this dep now?

Reply:
can probably drop mocha and the test script as well, I know it's orthogonal to this change but since you're touching the file I figured I'd mention it